grass/grass.c

```diff
@@ -31,9 +33,13 @@ int main() {
     grass->sys_exit = sys_exit;
     grass->sys_send = sys_send;
     grass->sys_recv = sys_recv;
+    grass->sys_wait = sys_wait;
 
     /* Register the kernel entry */
     earth->kernel_entry_init(kernel_entry);
+
+    /* Initialize Kernel Buffer */
+    kernel_init();
```

Comment:
Is there a reason to add a function call here instead of doing the two-line initialization here directly?